docs/pages/admin-guides/infrastructure-as-code/teleport-operator/trusted-cluster.mdx

```diff
@@ -0,0 +1,419 @@
+---
+title: Deploy Trusted Clusters using Kubernetes Operator
+description: Use Teleport's Kubernetes Operator to deploy Trusted Clusters
+---
+
+<Admonition type="note">
+Trusted clusters are only available for self-hosted Teleport clusters.
+</Admonition>
+
+This guide will explain how to use Teleport's Kubernetes Operator to deploy
+trusted clusters.
+
+## How it works
+
+The Teleport Kubernetes Operator provides a way for Kubernetes users to manage Teleport trusted clusters through
+Kubernetes, following [the Operator Pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+The trusted cluster custom resource definitions are deployed alongside the Teleport
+Kubernetes Operator. Once deployed, users can use a Kubernetes client like `kubectl`
+or their existing CI/CD Kubernetes pipelines to create and manage Teleport trusted
+clusters. The Teleport Kubernetes Operator watches for these trusted cluster resources
+and makes API calls to Teleport to reach the desired state.
+
+## Prerequisites
+
+- Access to **two** Teleport cluster instances.
```

Comment:
I would clarify that we expect each cluster to be on a Linux server, then link to the [Linux demo guide](https://goteleport.com/docs/admin-guides/deploy-a-cluster/linux-demo/). Otherwise, I didn't know "instances" meant "VM instances" until I got a couple items into Step 1.